generator-api/src/main/scala/com/lightbend/lagom/spec/ResourceUtils.scala

```diff
@@ -42,4 +43,25 @@ object ResourceUtils {
     }
   }
 
+  /**
+   * @param folder an absolute folder
+   * @param filename a relative path to a file (may contain File.separators)
+   * @param fileContents
+   * @return
+   */
+  def writeFile(folder: File, filename: String, fileContents: String): File = {
+    val path = Paths.get(folder.getAbsolutePath, filename)
+    val theFolder = path.toFile.getParentFile
+    while (!theFolder.exists()) {
+      theFolder.mkdirs()
```

Comment:
Maybe it's better to use [`Files.createDirectories`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createDirectories-java.nio.file.Path-java.nio.file.attribute.FileAttribute...-) than `mkdirs` in a loop.